multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/flowable/commands/ClearJobLockOwnersCmd.java

```diff
@@ -0,0 +1,44 @@
+package org.cloudfoundry.multiapps.controller.process.flowable.commands;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.flowable.common.engine.impl.interceptor.Command;
+import org.flowable.common.engine.impl.interceptor.CommandContext;
+import org.flowable.job.api.Job;
+import org.flowable.job.service.JobServiceConfiguration;
+import org.flowable.job.service.impl.persistence.entity.JobEntity;
+import org.flowable.job.service.impl.persistence.entity.JobEntityManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ClearJobLockOwnersCmd implements Command<Void> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClearJobLockOwnersCmd.class);
+
+    private final List<Job> lockedJobs;
+    private final JobServiceConfiguration jobServiceConfiguration;
+
+    public ClearJobLockOwnersCmd(List<Job> lockedJobs, JobServiceConfiguration jobServiceConfiguration) {
+        this.lockedJobs = lockedJobs;
+        this.jobServiceConfiguration = jobServiceConfiguration;
+    }
+
+    @Override
+    public Void execute(CommandContext commandContext) {
+        JobEntityManager jobEntityManager = jobServiceConfiguration.getJobEntityManager();
+        lockedJobs.forEach(job -> {
```

Comment:
I think it would be cleaner to use a regular for loop.
The main use case for the `Collection.forEach` method is, in my opinion, for one-liners:
```java
list.forEach(this::doSomething);
```
Otherwise, a regular for loop is cleaner and easier to read